package.json

```diff
@@ -20,14 +20,14 @@
     "clean-dist": "rimraf dist",
     "clean-lib": "rimraf lib",
     "clean": "builder run clean-lib && builder run clean-dist",
-    "build-dist-min": "webpack --bail --config node_modules/builder-victory-component/config/webpack/webpack.config.js --colors",
-    "build-dist-dev": "webpack --bail --config node_modules/builder-victory-component/config/webpack/webpack.config.dev.js --colors",
+    "build-dist-min": "BABEL_ENV=webpack webpack --bail --config node_modules/builder-victory-component/config/webpack/webpack.config.js --color",
```

Comment:
Will break on windows. You can't set environment variables this way. (however you _can_ do `process.env.BABEL_ENV = "webpack"` in these configuration files themselves.